### ISSUE TITLE
Bridgy Fed website: random-ish order of events on profile page

### ISSUE THREAD

**Issue by mackuba** [snarfed/bridgy-fed#1012]
https://fed.brid.gy/bsky/mackuba.eu

Events are shown with timestamps:
1...5 hours ago
18 hours ago
7 hours ago
18 hours ago
2 days ago
1 day ago
7 days ago
2 days ago
8 days ago
...

**Comment by snarfed** [comment]
Yeah this is expected right now, they're sorted by last update time but shown with created time, but still confusing, should fix one way or another.
